internal/services/user/user.go

```diff
@@ -97,11 +119,17 @@ func (s Service) ListUsers(ctx context.Context, req *authd.Empty) (*authd.Users,
 // GetGroupByName returns the group entry for the given group name.
 func (s Service) GetGroupByName(ctx context.Context, req *authd.GetGroupByNameRequest) (*authd.Group, error) {
 	if req.GetName() == "" {
+		log.Warningf(ctx, "GetGroupByName: no group name provided")
 		return nil, status.Error(codes.InvalidArgument, "no group name provided")
 	}
 
 	g, err := s.userManager.GroupByName(req.GetName())
+	if errors.Is(err, users.NoDataFoundError{}) {
+		// Only log this at debug level, see GetUserByName for details
+		log.Debugf(context.Background(), "GetGroupByName: %v", err)
+	}
```

Comment:
If we are adding this every single time after those calls, maybe we should consider moving this log message inside the GroupByName function instead? (same thing for the other calls as well)